src/components/structures/MessagePanel.tsx

```diff
@@ -274,6 +279,10 @@ export default class MessagePanel extends React.Component<IProps, IState> {
         }
     }
 
+    private calculateRoomMembersCount = (): void => {
+        this.membersCount = this.props.room?.getMembers().length || 0;
+    };
```

Comment:
this doesn't setState so will have a very jarring update behaviour, it'll take effect the render after this fires, weirdly laggy. I don't think this is the right solution and think it does not whatsoever match what I accepted in review. @gsouquet